xla/service/gpu/runtime/BUILD

```diff
@@ -25,11 +24,27 @@ package_group(
     ],
 )
 
+cc_library(
+    name = "gpu_kernel_helper",
+    hdrs = if_gpu_is_configured(["gpu_kernel_helper.h"]),
+    # we use 'defines' in order to propagate GPU platform flags to dependent
+    # libraries
+    defines = if_cuda_is_configured(["GOOGLE_CUDA=1"]) + 
```

Comment:
Using defines is not a good choice if we can avoid it. Better to use local_defines for the targets that need it.

Reply:
ok, sure, if it's undersired, I can replace it with local_defines